### DISCUSSION TITLE
又一个 zig-time

### DISCUSSION THREAD

**Discussion by deatil** [zigcc/forum#126] [作品分享]
众所周知，zig现在基本是只能获取时间戳和部分时间的格式化，所以时间处理的库也有好些。
但是大多不太符合自己的预期，就开工自己写了一个。
部分函数逻辑来自go，格式化format想法来自于js的时间库。本地时区获取暂时处于feture阶段。
欢迎使用star。
项目地址: https://github.com/deatil/zig-time

**Comment by jiacai2050** [comment]
可以添加到这里 https://github.com/zigcc/awesome-zig?tab=readme-ov-file#date-time-and-timezones

**Reply by deatil** [reply]
添加了
